ui/pages/token-details/token-details-page.js

```diff
@@ -182,15 +191,17 @@ export default function TokenDetailsPage() {
           <>
             <Typography
               variant={TYPOGRAPHY.H9}
-              margin={[4, 0, 0, 0]}
+              margin={0}
+              marginTop={4}
               color={COLORS.TEXT_ALTERNATIVE}
               fontWeight={FONT_WEIGHT.BOLD}
             >
               {t('tokenList')}
             </Typography>
             <Typography
               variant={TYPOGRAPHY.H7}
-              margin={[1, 0, 0, 0]}
+              margin={0}
+              marginTop={1}
               color={COLORS.TEXT_DEFAULT}
             >
               {`${aggregators}.`}
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props